UnitedIncome/components
# Add UmbrellaCircleIllustration and DocumentCircleIllustration to component library

### Description
> Please provide a brief description of your changes below.

Illustrations are already added, but were not exported. This fixes that.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

N/A

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.
